app/src/main/java/com/wednesday/banklocator/model/IfscResponse.kt

```diff
@@ -0,0 +1,13 @@
+package com.wednesday.banklocator.model
+
+data class IfscResponse(
+    val ADDRESS: String,
+    val BANK: String,
+    val BRANCH: String,
+    val CITY: String,
+    val CONTACT: String,
+    val DISTRICT: String,
+    var IFSC: String,
+    val RTGS: Boolean,
+    val STATE: String

```

Comment:
Why is everything capital?

Reply:
Some of the files including this "data class" are auto-generated using the ".json to class object" plugin.

Reply:
I'll change them.